package.json

```diff
@@ -20,14 +20,18 @@
   },
   "homepage": "https://github.com/RisingStack/bootcamp-roland#readme",
   "dependencies": {
-    "koa": "^2.13.0",
-    "koa-router": "^9.4.0"
+    "graphql": "15.3.0",
+    "graphql-request": "3.1.0",
+    "koa": "2.13.0",
+    "koa-router": "9.4.0"
   },
   "devDependencies": {
-    "eslint": "7.8.1",
-    "mocha": "8.1.3",
     "chai": "4.2.0",
     "chai-http": "4.3.0",
+    "dotenv-safe": "^8.2.0",
```

Comment:
I think it should be a normal dependency with version fixed :stuck_out_tongue_winking_eye: